Makefile

```diff
@@ -5,6 +5,9 @@ STABLE_TARGETS = $(shell hack/chart_destination.sh $(STABLE_CHARTS))
 STAGING_CHARTS = $(wildcard staging/*/Chart.yaml)
 STAGING_TARGETS = $(shell hack/chart_destination.sh $(STAGING_CHARTS))
 
+GIT_REMOTE_URL := $(shell git remote get-url origin)
+GITHUB_USER := $(shell git remote get-url origin | cut -d':' -f2 | cut -d'/' -f1)
```

Comment:
This line return an empty space in my case. Please, fix it and open a different PR cause the changes here and in the title of the PR target different goals.

Reply:
:thinking: Thanks for checking - I've moved that commit to https://github.com/mesosphere/charts/pull/10 and will drop it from this PR.